Vagrantfile

```diff
@@ -102,7 +102,7 @@ Vagrant.configure('2') do |config|
     config.vm.synced_folder synced_folder.fetch('local_path'), synced_folder.fetch('destination'), options
   end
 
-  config.vm.provision provisioner do |ansible|
+  config.vm.provision 'drupalvm', type: 'ansible' do |ansible|
```

Comment:
You should keep using the `provisioner` variable for `type` as it supports both ansible and ansible_local.

Reply:
Didn't know that! So like this?

```
config.vm.provision 'drupalvm', type: 'provisioner' do |ansible|
```

Reply:
```rb
config.vm.provision 'drupalvm', type: provisioner do |ansible|
```

Earlier on in the `Vagrantfile` the variable is set (`vagrant_provisioner` is a function in lib/drupalvm/vagrant.rb):

```rb
provisioner = vconfig['force_ansible_local'] ? :ansible_local : vagrant_provisioner
```

Reply:
Gotcha.